articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+Overall Silo is taking security seriously and has taken the necessary measures to circumvent vulnerabilities. However, both audits were limited to Silo’s lending platform before XAI was introduced. It can be argued that XAI is a standard mint/burn contract controlled by the DAO that doesn’t need an audit, however, it must be highlighted nonetheless as a potential risk.
+
+
+## Depeg Risk
+
+XAI is a new stablecoin. There is no historic data about its price stability or behavior during highly volatile market conditions. We can, however, presume that Silo has and will implement all the right measures to ensure XAI’s stability.
+
+As with other stablecoins, keeping the peg depends on a few key factors:
+
+
+
+1. The underlying collateral needs to be fit for purpose (i.e. highly volatile collateral increases the risk of default and the risk of liquidations)
+2. For the market to absorb liquidations, as well as arbitrage to keep the peg, both the stablecoin and the collateral asset need to have deep enough liquidity
+3. The protocol can support price stability via parameter adjustments
+4. Ideally, the protocol has some fallback solution in case of a black swan event
+
+We’ll discuss the collateral risk (1) in more detail in the next section. The other stabilization mechanisms highlighted above are mostly covered:
+
+
+
+* XAI is an over-collateralized stablecoin. At first only USDC and ETH will serve as collateral for XAI
+* Arbitrage possibilities are given and liquidity in the open market will be seeded by the DAO and incentivized through CVX gauges (see [proposal](https://gov.silo.finance/t/building-on-chain-liquidity-for-xai/309))
+* The DAO influences XAI via parameter adjustments, such as borrow rate, adding/removing collateral, increasing/decreasing XAI availability
+
+The only measure that is missing is a concrete fallback solution, that the DAO can execute in case things go south (e.g. use SILO as a backup to stabilize the stablecoin). However, this can also come at a later stage, once XAI has achieved some product market fit.
+
+The last thing to highlight related to stability is that Silo’s TVL is currently around ~$1M. More than half of that TVL was [seeded by the DAO](https://snapshot.org/#/silofinance.eth/proposal/0x013bbb154af9317ab7a78b72a498d16e069fdba5e78b710e6785fc938d644e76) itself. There needs to be significant growth in TVL to enable sufficient backing of a stablecoin that aims to facilitate bridging between 60+ assets (10 assets are already live and [57 assets](https://gov.silo.finance/t/deploying-57-new-silos-to-the-protocol/306/2) will be added soon). While there are plans to incentivize liquidity for XAI, there is currently no plan presented that defines anything specific about how the DAO plans to attract more TVL for its long-tail asset silos (a potential [veTokenomics](https://gov.silo.finance/t/tokenomics-proposal-vesilo-v2/226) discussion seems to be stuck).
```

Comment:
TVL jumped to 3m today, looks like influx of USDC, just FYI, still small numbers anyways